reference/forms/types/choice.rst

```diff
@@ -230,7 +231,13 @@ correct types will be assigned to the model.
 
 .. include:: /reference/forms/types/options/choice_name.rst.inc
 
+<<<<<<< HEAD
```

Comment:
Can you please solve the merge conflict? Thanks

Reply:
Solved